src/dodal/devices/turbo_slit.py

```diff
@@ -1,8 +1,9 @@
-from ophyd_async.core import StandardReadable
+from bluesky.protocols import Movable, Readable
+from ophyd_async.core import AsyncStatus
 from ophyd_async.epics.motor import Motor
 
 
-class TurboSlit(StandardReadable):
+class TurboSlit(Movable, Readable):
```

Comment:
Must: This is throwing type errors in `i20_1` as it needs to be a `HasName` and implement `read`, additionally it means that . I think it probably needs to be:
```suggestion
class TurboSlit(StandardReadable, Movable):
```